### PR TITLE
Nolight snow tiles

### PR DESCRIPTION
**Pull request by MadmanMartian**
something for plosky

### CODE REVIEW

File: code/game/turfs/simulated/snow.dm [hunk 1]
```diff
@@ -13,6 +13,10 @@
 	var/snowballs = 0
 	var/global/list/icon_state_to_appearance = list()
 
+/turf/simulated/floor/plating/snow/nolight
+	dynamic_lighting = 1
+	light_color = ""
```
Comment (PJB3005) [inline code/game/turfs/simulated/snow.dm]: Why this?

**Comment by SonixApache** [comment]
might as well just make simsnow dynamically lit, would make away missions more interesting as well as possibly solving the TILE OF DARKNESS issues

**Comment by MadmanMartian** [comment]
alright, I'll do that

**Comment by plosky1** [comment]
Can you also do the same for the unsimmed ones, upon talking with PJB and burneddi it should all be unsimmed apparently.

**Comment by brndd** [comment]
Just make the existing unsim snow (`/turf/unsimulated/floor/snow`) dynamically lit too, the only places that use it are the vaults/away missions @SonixApache mentioned.

**Comment by SonixApache** [comment]
now certain vaults are going to be suddenly unlit which clashes with other vaults, but okay

**Pull request merged**